src/serenity/config.hpp

```diff
@@ -55,42 +72,6 @@ class SerenityConfig {
     return *getSection(key);
   }
 
-  // -- unsafe getters --
-
-  /**
-   * Unsafe getter for string
-   */
-  std::string getS(std::string key) {
-    return boost::get<std::string>(this->fields[key]);
-  }
-
-  /**
-   * Unsafe getter for int64_t
-   */
-  int64_t getI64(std::string key) {
-    return boost::get<int64_t>(this->fields[key]);
-  }
-
-  /**
-   * Unsafe getter for uint64_t
-   */
-  uint64_t getU64(std::string key) {
-    return boost::get<uint64_t>(this->fields[key]);
-  }
-
-  /**
-   * Unsafe getter for double_t
-   */
-  double_t getD(std::string key) {
-    return boost::get<double_t>(this->fields[key]);
-  }
-
-  /**
-   * Unsafe getter for bool
-   */
-  bool getB(std::string key) {
-    return boost::get<bool>(this->fields[key]);
-  }
 
   // -- setters --
 
```

Comment:
It might be hard to distinguish uint64_t from int64_t when parsing json... We might rather use just int64_t.